sys/net/gnrc/network_layer/ipv6/nib/_nib-6lr.c

```diff
@@ -84,8 +84,16 @@ uint8_t _reg_addr_upstream(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
 #endif  /* CONFIG_GNRC_IPV6_NIB_MULTIHOP_DAD */
             if (aro->ltime.u16 != 0) {
                 _handle_sl2ao(netif, ipv6, icmpv6, sl2ao);
+
                 /* re-get NCE in case it was updated */
                 nce = _nib_onl_get(&ipv6->src, netif->pid);
+
+                /* NIB is full */
+                /* TODO: evict oldest entry */
+                if (nce == NULL) {
+                    return _ADDR_REG_STATUS_IGNORE;
```

Comment:
```suggestion
                    return SIXLOWPAN_ND_STATUS_NC_FULL;
```

So that it is properly reported to the downstream node?

see

https://github.com/RIOT-OS/RIOT/blob/18062aca09722475819366da2ccf2f4e377fb61b/sys/net/gnrc/network_layer/ipv6/nib/_nib-6ln.c#L159

https://github.com/RIOT-OS/RIOT/blob/18062aca09722475819366da2ccf2f4e377fb61b/sys/net/gnrc/network_layer/ipv6/nib/_nib-6lr.c#L134-L146